app/wedding/serializers.py

```diff
@@ -1,8 +1,15 @@
 from rest_framework import serializers
 from core.models import Wedding
+from core.models import Guest
 
 
 class WeddingSerializer(serializers.ModelSerializer):
     class Meta:
         model = Wedding
         fields = ('id', 'name', 'email', 'date', 'image')
+
+
+class GuestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Guest
+        fields = ('id', 'name', 'phone_number', 'wedding')
```

Comment:
Great work with the serializers in this file. And good to know that wedding needed to be included in the fields in order to complete the connection between wedding and guest

Reply:
What an annoying thing to miss for 24 hours! :P